modules/gridmap/editor/grid_map_editor_plugin.cpp

```diff
@@ -1204,6 +1204,7 @@ GridMapEditor::GridMapEditor() {
 	ED_SHORTCUT("grid_map/fill_selection", TTR("Fill Selection"), KeyModifierMask::CTRL + Key::F);
 
 	int mw = EDITOR_DEF("editors/grid_map/palette_min_width", 230);
+	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::INT, "editors/grid_map/palette_min_width", PROPERTY_HINT_RANGE, "100,600,1"));
```

Comment:
Checking the code and this value is in pixels. However it is multiplied by the editor scale. I would personally still add the suffix but I'm not sure how other properties handle this? 
For some reason `preview_size` isn't in pixels for sure. The whole **GridMap** code does smell old so it's understandable.